src/test/java/uk/gov/hmcts/darts/arm/component/impl/ArmResponseFilesProcessSingleElementImplTest.java

```diff
@@ -1096,6 +1104,6 @@ void processResponseFilesFor_WithValidCreateRecordFileAndValidUploadFileSuccess(
         verify(armDataManagementApi).deleteBlobData(createRecordFileFilename);
         verifyNoMoreInteractions(armDataManagementApi);
 
-        verify(logApi).archiveToArmSuccessful(anyInt());
+        verify(logApi, never()).archiveToArmSuccessful(anyInt());
```

Comment:
Why has this changed to never?

Reply:
Because in the code that log happens only if the status is STORED, and @hemantasharma1129 asked to leave that exactly as it is now. Therefore now when on the intermediate status RPO_PENDING that log won't happen

Reply:
Cool thank you for confirming :)